### ISSUE TITLE
Create a new page with info about licensing/Creative Commons etc

### ISSUE THREAD

**Issue by brycepj** [brycepj/MT_demo#4]
As per #3. 

We need to create a new page that has x, y, z as well as a, b, c. 

Which menu do you best think this fits under?

**Comment by winndixiebagofmoney** [comment]
I think it should under ABC menu, and that the page should have lots of pretty pictures and words.

**Comment by brycepj** [comment]
I agree -- I should be able to get this done pretty quickly. Assigning it to myself.
